client/src/components/Navbar.js

```diff
@@ -105,6 +116,8 @@ function getNavigation({ width, classes, anchorEl, handleClick, handleClose }) {
 function Navbar(props) {
   const { user } = useUserContext();
   const classes = useStyles();
+  const { profile } = useProfileContext();
+  console.log(profile)
```

Comment:
extra logging here

Reply:
Oops, missed this one when I was remove console logs, just pushed a new commit